container/ubuntu16_04/layers/gcloud/gcloud.yaml

```diff
@@ -0,0 +1,7 @@
+schemaVersion: "2.0.0"
+
+commandTests:
+- name: 'check-gcloud'
+  command: 'gcloud'
+  args: ['version']
+  expectedOutput: ['Google Cloud SDK.*']
```

Comment:
Can you create a bzl file with the gcloud sha256 + version string and use a tpl to generate a yaml file that checks the version (and replace use of sha in WORKSPACE for use of bzl)

Reply:
done

The sha in WORKSPACE is for the pgp key, which doesn't change normally.